packages/jsii-calc/lib/compliance.ts

```diff
@@ -1335,3 +1335,34 @@ export class Constructors {
         return new PrivateClass();
     }
 }
+
+// internal can be used to represent members that can only be accessed from the current module
+export class StripInternal {
+    public youSeeMe = 'hello';
+
+    /**
+     * This is an internal thing
+     * @internal
+     */
+    public youDontSeeMeAlthoughIamPublic = 'world'
+}
+
+/**
+ * @internal
+ */
+export class InternalClass {
+    public iAmNotHere = 'yes';
+}
+
+/**
+ * @internal
+ */
+export interface IInternalInterface { }
```

Comment:
What happens when I implement this `IInternalInterface` from a non-`@internal` type? I feel there's a host of missing test cases here that I suspect will blow up because the changes in the compiler itself are relatively minor in comparison to what I had expected to find...

Reply:
Addressed.